console/console_test.go

```diff
@@ -16,327 +16,328 @@
-func TestExecute(t *testing.T) {
-	tester := newTester(t, nil)
-	defer tester.Close(t)
-
-	tester.console.Execute("exec.js")
-
-	tester.console.Evaluate("execed")
-	if output := tester.output.String(); !strings.Contains(output, "some-executed-string") {
-		t.Fatalf("execed variable missing: have %s, want %s", output, "some-executed-string")
-	}
-}
-
-// Tests that the JavaScript objects returned by statement executions are properly
-// pretty printed instead of just displaying "[object]".
-func TestPrettyPrint(t *testing.T) {
-	tester := newTester(t, nil)
-	defer tester.Close(t)
-
-	tester.console.Evaluate("obj = {int: 1, string: 'two', list: [3, 3, 3], obj: {null: null, func: function(){}}}")
-
-	// Define some specially formatted fields
-	var (
-		one   = jsre.NumberColor("1")
-		two   = jsre.StringColor("\"two\"")
-		three = jsre.NumberColor("3")
-		null  = jsre.SpecialColor("null")
-		fun   = jsre.FunctionColor("function()")
-	)
-	// Assemble the actual output we're after and verify
-	want := `{
-  int: ` + one + `,
-  list: [` + three + `, ` + three + `, ` + three + `],
-  obj: {
-    null: ` + null + `,
-    func: ` + fun + `
-  },
-  string: ` + two + `
-}
-`
-	if output := tester.output.String(); output != want {
-		t.Fatalf("pretty print mismatch: have %s, want %s", output, want)
-	}
-}
-
-// Tests that the JavaScript exceptions are properly formatted and colored.
-func TestPrettyError(t *testing.T) {
-	tester := newTester(t, nil)
-	defer tester.Close(t)
-	tester.console.Evaluate("throw 'hello'")
-
-	want := jsre.ErrorColor("hello") + "\n\tat <eval>:1:7(1)\n\n"
-	if output := tester.output.String(); output != want {
-		t.Fatalf("pretty error mismatch: have %s, want %s", output, want)
-	}
-}
-
-// Tests that tests if the number of indents for JS input is calculated correct.
-func TestIndenting(t *testing.T) {
-	testCases := []struct {
-		input               string
-		expectedIndentCount int
-	}{
-		{`var a = 1;`, 0},
-		{`"some string"`, 0},
-		{`"some string with (parenthesis`, 0},
-		{`"some string with newline
-		("`, 0},
-		{`function v(a,b) {}`, 0},
-		{`function f(a,b) { var str = "asd("; };`, 0},
-		{`function f(a) {`, 1},
-		{`function f(a, function(b) {`, 2},
-		{`function f(a, function(b) {
-		     var str = "a)}";
-		  });`, 0},
-		{`function f(a,b) {
-		   var str = "a{b(" + a, ", " + b;
-		   }`, 0},
-		{`var str = "\"{"`, 0},
-		{`var str = "'("`, 0},
-		{`var str = "\\{"`, 0},
-		{`var str = "\\\\{"`, 0},
-		{`var str = 'a"{`, 0},
-		{`var obj = {`, 1},
-		{`var obj = { {a:1`, 2},
-		{`var obj = { {a:1}`, 1},
-		{`var obj = { {a:1}, b:2}`, 0},
-		{`var obj = {}`, 0},
-		{`var obj = {
-			a: 1, b: 2
-		}`, 0},
-		{`var test = }`, -1},
-		{`var str = "a\""; var obj = {`, 1},
-	}
-
-	for i, tt := range testCases {
-		counted := countIndents(tt.input)
-		if counted != tt.expectedIndentCount {
-			t.Errorf("test %d: invalid indenting: have %d, want %d", i, counted, tt.expectedIndentCount)
-		}
-	}
-}
+//
```

Comment:
please revert this

Reply:
I will revert this after merging Ethereum patches into the develop branch, I comment on this in order to pass CI temporarily, we can keep this comment open until we finished the rebase work from the upstream develop branch.